stream-chat-android-ui-components/src/main/res/layout/stream_message_input.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/stream_white">
+
+    <LinearLayout
```

Comment:
`android:layout_width="0dp"` + `:app:layout_constraintXXX`

Reply:
We will probably add more views inside the ConstraintLayout, that's why I encapsulated this one inside the LinearLayout... anyways, it is better to just use the ConstraintLayout.